hudi-cli/src/main/java/org/apache/hudi/cli/commands/CommitsCommand.java

```diff
@@ -314,6 +314,45 @@ public String showCommitPartitions(
         limit, headerOnly, rows, exportTableName);
   }
 
+  @CliCommand(value = "commit show_commit_write_stats", help = "Show write stats of a commit")
```

Comment:
extreme nit: s/show_commit_write_stats/show_write_stats since we already know this is a commits command.